.github/workflows/publish-layoutlib.yml

```
@@ -0,0 +1,35 @@
+name: publish-layoutlib
+
+on:
+  workflow_dispatch:
+
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+
+jobs:
+
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Configure JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version: 17
+
+      - name: Publish Artifacts
+        run: ./gradlew publishMavenPublicationToMavenCentralRepository
```

```suggestion
        run: ./gradlew publishMavenNativeLibraryPublicationToMavenCentralRepository
```
otherwise, this would publish a partial release of Paparazzi.

---

yep I had done the separate module deploys but that is way more concise. Updating